detect_variants/detect_variants.cwl

```diff
@@ -11,32 +11,28 @@ inputs:
         secondaryFiles: [".fai", "^.dict"]
     tumor_bam:
         type: File
-        secondaryFiles: [".bai"]
+        secondaryFiles: [.bai]
```

Comment:
You will likely have merge conflicts here.  Since GATK doesn't produce a .bam.bai file extension you will want to use `[^.bai]`